src/Microsoft.DotNet.Build.VstsBuildsApi/VstsReleaseHttpClient.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Build.VstsBuildsApi
+{
+    internal class VstsReleaseHttpClient : IVstsDefinitionHttpClient
+    {
+        private static readonly string[] s_InstanceIdentifiableField = {
+            "_links",
+            "createdBy",
+            "createdOn",
+            "deployStep",
+            "modifiedBy",
+            "modifiedOn",
+            "owner",
+            "id",
+            "lastRelease",
+            "revision",
+        };
```

Comment:
I think it makes sense to make this configurable, but if we need a new API version I'd think there would be more changes than just the array and version strings so I don't know if just a file makes sense for that scenario.